adafruit_scd30.py

```diff
@@ -76,7 +78,14 @@ def __init__(self, i2c_bus, ambient_pressure=0, address=SCD30_DEFAULT_ADDR):
     def reset(self):
         """Perform a soft reset on the sensor, restoring default values"""
         self._send_command(_CMD_SOFT_RESET)
-        sleep(0.030)  # not mentioned by datasheet, but required to avoid IO error
+        sleep(0.1)  # not mentioned by datasheet, but required to avoid IO error
+        # pylint:disable=protected-access
+        # are we using Blinka?
+        if hasattr(self.i2c_device.i2c, "_i2c"):
```

Comment:
id kinda sorta prefer not having this here but instead in a new simple example just for mcp2221 if thats ok?

Reply:
we'd have to remove the call to `reset()` in `__init__`, which is maybe OK? otherwise you can't even instantiate with mcp2221.

Reply:
>.<
ridiculous sensor. 
ok move the `reset()` out of init and into the example too